core/src/main/java/org/apache/carbondata/core/statusmanager/SegmentStatusManager.java

```diff
@@ -874,9 +874,9 @@ private ValidAndInvalidSegmentsInfo(List<Segment> listOfValidSegments,
   }
 
   /**
-   * Return true if any load or insert overwrite is in progress for specified table
+   * Return true if insert or insert overwrite is in progress for specified table
    */
-  public static Boolean isLoadInProgressInTable(CarbonTable carbonTable) {
+  public static boolean isInsertInProgress(CarbonTable carbonTable) {
```

Comment:
if you change the method name from 'load' to 'insert', you should change the return parameter name the same time.
-> return loadInProgress to return insertInProgress

Reply:
you are right,  I have changed the method to "isTableInLoadProgress"